From ac15caa31054ad166d82c3228a05bd0c5dff0799 Mon Sep 17 00:00:00 2001
From: Saulius Archipovas <ich@sauli.de>
Date: Mon, 5 Mar 2012 18:49:17 +0100
Subject: [PATCH 2/2] forgot group.rb

---
 app/models/group.rb |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/app/models/group.rb b/app/models/group.rb
index a59a0ce..94fa51b 100644
--- a/app/models/group.rb
+++ b/app/models/group.rb
@@ -2,5 +2,5 @@ class Group < ActiveRecord::Base
 	has_many :users
 	belongs_to :project
 
-	validates_associated :users
+	validates_presence_of :users
 end
-- 
1.7.9

